pkg/platform/client.go

```diff
@@ -106,7 +106,9 @@ type VirtualClusterInstanceProject struct {
 	Project        *managementv1.Project
 }
 
-func NewClientFromConfig(ctx context.Context, config *config.CLI) (Client, error) {
+// InitClientFromConfig returns a client with the client identity initialized through the selves api.
+// Use this by default, unless performing actions that don't require a log in (like login itself).
+func InitClientFromConfig(ctx context.Context, config *config.CLI) (Client, error) {
 	c := &client{
 		config: config,
 	}
```

Comment:
Couldn't we call `NewClientFromConfig` here instead of manually initializing the struct?